core/src/Authorization.html

```diff
@@ -1,88 +1,131 @@
       on:click="{onUserSettingsClick}"
-      data-testid="{getTestId(profileNav.settings)}"
     >
-      <a aria-label="User Settings" class="fd-menu__link" data-testid="settings-link">
+      <a
+        tabindex="0"
+        class="fd-list__link"
+        aria-label="User Settings"
+        data-testid="settings-link"
+      >
         {#if profileNav.settings.icon} {#if hasOpenUIicon(profileNav.settings)}
-        <i class="fd-top-nav__icon sap-icon--{profileNav.settings.icon}"></i>
+        <i
+          role="presentation"
+          class="fd-list__icon sap-icon--{profileNav.settings.icon}"
+        ></i>
         {:else}
         <img
           class="fd-top-nav__icon nav-icon"
           src="{profileNav.settings.icon}"
           alt="{profileNav.settings.altText ? profileNav.settings.altText : ''}"
         />
-        {/if} {/if}
-        <span class="fd-menu__title">{$getTranslation(profileNav.settings.label)}</span>
-      </a>
-    </li>
-    {/if} {#if isAuthorizationEnabled || isProfileLogoutItem} {#if isLoggedIn ||
-    !isAuthorizationEnabled && isProfileLogoutItem}
-    <li
-      class="fd-menu__item"
-      on:click="{onLogoutClick}"
-      data-testid="{getTestId(profileNav.logout)}"
-    >
-      <a aria-label="Logout" class="fd-menu__link" data-testid="logout-link">
-        {#if profileNav.logout.icon} {#if hasOpenUIicon(profileNav.logout)}
-        <i class="fd-top-nav__icon sap-icon--{profileNav.logout.icon}"></i>
-        {:else}
-        <img
-          class="fd-top-nav__icon nav-icon"
-          src="{profileNav.logout.icon}"
-          alt="{profileNav.logout.altText ? profileNav.logout.altText : ''}"
-        />
-        {/if} {/if}
-        <span class="fd-menu__title">{$getTranslation(profileNav.logout.label)}</span>
+        {/if}{/if}
+        <span class="fd-list__title">{$getTranslation(profileNav.settings.label)}</span>
       </a>
     </li>
-    {/if} {#if isAuthorizationEnabled && !isLoggedIn}
-    <li class="fd-menu__item" on:click="{startAuthorization}">
-      <a aria-label="Login" class="fd-menu__link" data-testid="login-link">
-        <span class="fd-menu__title">Login</span>
-      </a>
-    </li>
-    {/if} {/if}
+    {/if}
   </ul>
-</nav>
+  {/if}
+</div>
+<div class="fd-popover__body-footer">
+  <div class="fd-bar fd-bar--footer">
+    <div class="fd-bar__right">
+      <div class="fd-bar__element">
+        {#if isAuthorizationEnabled || isProfileLogoutItem} {#if isLoggedIn ||
+        !isAuthorizationEnabled && isProfileLogoutItem}
+        <div class="fd-bar__element" data-testid="{getTestId(profileNav.logout)}">
+          <button
+            class="fd-button fd-button--transparent"
+            tabindex="0"
+            on:click="{onLogoutClick}"
+            data-testid="logout-btn"
+          >
+            {$getTranslation(profileNav.logout.label)}
+          </button>
+        </div>
+        {/if}{/if} {#if isAuthorizationEnabled && !isLoggedIn}
```

Comment:
```suggestion
{/if} {#if isAuthorizationEnabled &amp;&amp; !isLoggedIn}        &lt;button          class="fd-button fd-button--transparent"          tabindex="0"          on:click="{startAuthorization}"          data-testid="login-btn"        &gt;          Login        &lt;/button&gt;        {/if}{/if}
```
I would move it inside this block like it was before.